docs/src/main/asciidoc/rabbitmq-reference.adoc

```diff
@@ -389,6 +389,19 @@ You need to indicate the name of the client using the `client-options-name` attr
 mp.messaging.incoming.prices.client-options-name=my-named-options
 ----
 
+== TLS Configuration
+
+RabbitMQ Messaging extension integrates with the xref:./tls-registry-reference.adoc[Quarkus TLS registry] to configure the Vert.x RabbitMQ client.
+
+To configure the TLS for a channel, you need to provide a named TLS configuration (or `<default>`, for the default configuration) in the `application.properties`:
```

Comment:
Following the ADR 0004 (https://github.com/quarkusio/quarkus/blob/main/adr/0004-using-the-tls-registry-for-clients.adoc) - it is recommended to use a named. So, I would remove the reference to `<default>`.